config/webpack.common.js

```diff
@@ -125,10 +130,58 @@ module.exports = function (options) {
          */
         {
           test: /\.scss$/,
-          use: ['to-string-loader', 'css-loader', 'sass-loader'],
+          use: [
+            'to-string-loader',
+            { loader: 'css-loader', options: { sourceMap, importLoaders: 1 } },
+            'postcss-loader',
+            {
+              loader: 'sass-loader',
+              options: {
+                sourceMap,
+                precision: 8,
+                includePaths: []
+              }
+            }
+          ],
           exclude: [helpers.root('src', 'styles')]
         },
 
+        /**
+         * To string and less loader support for *.less files (from Angular components)
+         * Returns compiled css content as string
+         *
+         */
+        {
+          test: /\.less$/,
```

Comment:
I get it. I simply added it so that users who prefer less can use it. The ideal would be to add it through some cli, but neither angular-cli does that, since this simply leaves it ready in the webpack.config for some future use.